pkgs/development/libraries/gtkd/default.nix

```diff
@@ -1,39 +1,24 @@
-{ lib, stdenv, fetchzip, fetchpatch, atk, cairo, ldc, gdk-pixbuf, gnome, gst_all_1, librsvg
+{ lib, stdenv, fetchzip, fetchpatch, atk, cairo, dcompiler, gdk-pixbuf, gnome, gst_all_1, librsvg
 , glib, gtk3, gtksourceview4, libgda, libpeas, pango, pkg-config, which, vte }:
 
 let
   inherit (gst_all_1) gstreamer gst-plugins-base gst-plugins-bad;
 in stdenv.mkDerivation rec {
   pname = "gtkd";
-  version = "3.9.0";
-
-  outputs = [ "out" "dev" ];
```

Comment:
We most likely want to keep this.

Reply:
```
error: cycle detected in build of '/nix/store/x6wh4v3hms7lnm0fm8xzxgh8y9ggi9m2-gtkd-3.10.0.drv' in the references of output 'dev' from output 'out'
```
Here's the full log:
[gtkd_761e87b336993b9fd7dd3c1bf64ac22162fad319.log](https://github.com/NixOS/nixpkgs/files/8657254/gtkd_761e87b336993b9fd7dd3c1bf64ac22162fad319.log)

Reply:
hmmm, I don't know how to properly debug this.